rllib/agents/impala/impala.py

```diff
@@ -101,12 +104,40 @@ class OverrideDefaultResourceRequest:
     def default_resource_request(cls, config):
         cf = dict(cls._default_config, **config)
         Trainer._validate_config(cf)
-        return Resources(
-            cpu=cf["num_cpus_for_driver"],
-            gpu=cf["num_gpus"],
-            extra_cpu=cf["num_cpus_per_worker"] * cf["num_workers"] +
-            cf["num_aggregation_workers"],
-            extra_gpu=cf["num_gpus_per_worker"] * cf["num_workers"])
+
+        eval_config = cf["evaluation_config"]
+
```

Comment:
Instead of converting, we could just have a if/else clause here to decide between old-style Resources and new-style PGF returns

Reply:
But that would mean a new config parameter. Don't we want to deprecate `Resources` anyways?

Reply:
So this is just to keep backwards compatibility. Once we deprecate `Resources` for good (in 1-2 releases) we will get rid of this. 
Let's not use a config parameter here, but rather an environment variable. Default behavior should be to use placement groups, but people should be able to opt out. I think you can re-use `TUNE_PLACEMENT_GROUP_AUTO_DISABLED` for this. Setting this to 1 would then mean that RLLibs default resources do not use Placement groups, and resource requests are not automatically converted. If someone deliberately sets this env variable to 1 and still want to use PGs in some cases they'll have to take care of that themselves. (I doubt that many people will want to mix anyways)